### DISCUSSION TITLE
Sync protocol documentation

### DISCUSSION THREAD

**Discussion by sagikazarmark** [GothenburgBitFactory/taskwarrior#2484] [General]
I'm trying to build something for taskwarrior using the sync protocol, but I couldn't find any documentation for the protocol.

According to #350, the protocol is documented in the taskserver repo, but I couldn't find where.

Right now, I'm trying to reverse engineer the protocol from the source code and some MITM hacks, but a documentation would make following changes much easier.

**Comment by tbabej** [comment]
I believe you're looking for this: https://taskwarrior.org/docs/design/protocol.html

**Reply by sagikazarmark** [reply]
Exactly, thanks!
